pkg/slock

```diff
@@ -1,20 +1,23 @@
 [mirrors]
-http://dl.suckless.org/tools/slock-1.1.tar.gz
+http://dl.suckless.org/tools/slock-1.4.tar.gz
 
 [vars]
-filesize=4604
-sha512=67cd9a62edde6ca0f9f7c09989ab8fa02a11a6ec82cc52b228c0b4fd509c5605668954630b0d2727aeb3c7283580665e4b5e5c0a37f3094a6f5b2a540e45f6b5
+filesize=6889
+sha512=ad285360dd3f16a225159abaf2f82fabf2c675bd74478cf717f68cbe5941a6c620e3c88544ce675ce3ff19af4bb0675c9405685e0f74ee4e84f7d34c61a0532f
 desc='locks your X11 session with your password'
-pkgver=3
+pkgver=4
 
 [deps]
 libx11
 
 [build]
-slockflags="-DVERSION=\"1.1\" -DHAVE_SHADOW_H -DCOLOR1=\"black\" -DCOLOR2=\"#005577\""
-$CC -D_GNU_SOURCE $optcflags $slockflags slock.c -o slock $optldflags -lX11 \
-   -Wl,-rpath-link=$butch_root_dir$butch_prefix/lib
+sed -i 's/nogroup/nobody/' config.def.h
 
-dest="$butch_install_dir""$butch_prefix"
-install -D -m 755 slock "$dest"/bin/slock
-chmod +s "$dest"/bin/slock
+make \
+  CFLAGS="$optcflags -DVERSION=\\\"1.4\\\" -D_DEFAULT_SOURCE -DHAVE_SHADOW_H" \
```

Comment:
you gotta pass CC too, otherwise X-compiles won't work

Reply:
nice catch